packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/delete/portabilities-delete.module.js

```diff
@@ -0,0 +1,25 @@
+import angular from 'angular';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-ui-angular';
+import 'ovh-api-services';
+
+import component from './portabilities-delete.component';
+import routing from './portabilities-delete.routing';
+
+const moduleName = 'ovhManagerTelecomTelephonyPortabilitiesDelete';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+    'oui',
+    'ovh-api-services',
+    'ngTranslateAsyncLoader',
+    'pascalprecht.translate',
+  ])
+  .config(routing)
+  .component('portabilitiesDelete', component)
```

Comment:
Component name might be too generic. We could probably update it later. Feel free to update it if needed. Thanks!